surefire-its/src/test/java/org/apache/maven/surefire/its/jiras/Surefire1787JUnit5IT.java

```diff
@@ -93,6 +93,18 @@ public void selectJUnit5()
                 "Using auto detected provider org.apache.maven.surefire.junitplatform.JUnitPlatformProvider" );
     }
 
+    @Test
+    public void selectJUnit5UsingConfiguredProvider()
+    {
+        unpack( "junit-4-5" )
+            .activateProfile( "select-junit5-using-configured-provider" )
+            .executeTest()
+            .verifyErrorFree( 1 )
+            .verifyTextInLog( "Running pkg.JUnit5Test" )
+            .verifyTextInLog(
+                "Using configured provider org.apache.maven.surefire.junitplatform.JUnitPlatformProvider" );
+    }
+
```

Comment:
Please remove this test, such we have separate one.
probably will fail - let's see.

Reply:
yes this is failing for me locally.
interesting it does not fail for gh action usign 18-ea. weird

Reply:
it is ok, first build is only fail-fast check without integration tests

Reply:
Doh - sorry! Thought I'd removed it already. Amended in e02e872921342459f81652ba6d9b64366fa093b2.

Reply:
no worries! thanks to fix it.